### PR TITLE
chore: tutorial changes

### LINKED ISSUE #301: Update tutorial styles
- Update titles per slide
- Remove gradient
- Update subtitle with instructions
- Remove overlay text on top of gifs

### PR DESCRIPTION
**Pull request by belohlavek**
Kapture 2019-03-12 at 10 43 27

Depends on https://github.com/decentraland/ui/pull/29

- Updates tutorial styles
- Adds last email step to tutorial
- FIll contest with tutorial email
- Tracks emails on mailchimp

Closes #301

### CODE REVIEW

**Comment by belohlavek** [comment]
Missing track email on contest submit

File: src/components/Modals/TutorialModal/TutorialModal.tsx [hunk 1]
```diff
@@ -1,38 +1,97 @@
...
+            type="email"
+            label="E-mail"
+            placeholder="you@your-email.com"
+            value={existingEmail || email}
+            onChange={this.handleEmailChange}
+            disabled={isLoading || !!existingEmail}
+            action={t('global.sign_up')}
+            onAction={this.handleSubmitEmail}
+            message={
+              existingEmail ? "Thanks! We'll stay in touch!" : 'By clicking on SIGN ME UP you are subscribing to news about our contest'
```
Comment (belohlavek) [inline src/components/Modals/TutorialModal/TutorialModal.tsx]: i18n

File: src/modules/translation/languages/en.json [hunk 2]
```diff
@@ -167,10 +168,28 @@
...
+      "title": "Arrows and orbits",
+      "description": "Rotate objects by clicking on the Rotate tool and dragging the orbits."
+    },
+    "slide3": {
+      "title": "Controlling the camera",
+      "description": "Drag with the left button to rotate the camera. Use the right button to pan."
+    },
+    "slide4": {
+      "title": "Participate in the contest",
+      "description": "Submit your scene to the contest using the \"Submit to contest\" button for a change to win LAND and MANA. Get notified of future updates:"
```
Comment (cazala) [inline src/modules/translation/languages/en.json]: for a ~change~ **chance** to win LAND and MANA

**Comment by nicosantangelo** [comment]
Let me know when this is ready @belohlavek !

File: src/modules/analytics/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { env } from 'decentraland-commons'
```
[the remark this answers is not in the document]
Comment (belohlavek) [reply src/modules/analytics/helpers.ts]: je

File: src/modules/analytics/helpers.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+export function reportEmail(email: string, interest: string) {
```
[the remark this answers is not in the document]
Comment (belohlavek) [reply src/modules/analytics/helpers.ts]: sure ;)

File: src/modules/analytics/helpers.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  return fetch(env.get('REACT_APP_MAIL_SERVER', ''), {
```
[the remark this answers is not in the document]
Comment (belohlavek) [reply src/modules/analytics/helpers.ts]: Let's do it

File: src/components/Modals/TutorialModal/TutorialModal.tsx [hunk 6]
```diff
@@ -1,38 +1,104 @@
...
-import { Button } from 'decentraland-ui'
+import { getAnalytics } from 'decentraland-dapps/dist/modules/analytics/utils'
+import { Button, Field } from 'decentraland-ui'
 import Modal from 'decentraland-dapps/dist/containers/Modal'
 import { getLocalStorage } from 'decentraland-dapps/dist/lib/localStorage'
 import { t, T } from 'decentraland-dapps/dist/modules/translation/utils'
 
 import { Props, State } from './TutorialModal.types'
 import './TutorialModal.css'
+import { reportEmail } from 'modules/analytics/helpers'
```
[the remark this answers is not in the document]
Comment (belohlavek) [reply src/components/Modals/TutorialModal/TutorialModal.tsx]: equis de

File: src/components/Modals/TutorialModal/TutorialModal.container.ts [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import { Dispatch } from 'redux'
+import { connect } from 'react-redux'
+
+import { RootState } from 'modules/common/types'
+import { MapStateProps, MapDispatchProps } from './TutorialModal.types'
+import TutorialModal from './TutorialModal'
```
[the remark this answers is not in the document]
Comment (belohlavek) [reply src/components/Modals/TutorialModal/TutorialModal.container.ts]: equis de

File: src/components/Modals/TutorialModal/TutorialModal.tsx [hunk 8]
```diff
@@ -49,15 +115,20 @@ export default class TutorialModal extends React.PureComponent<Props, State> {
     const slide = this.slides[step]
 
     return (
-      <div key={`slide-${step}`} className={'slide ' + slide.thumbnail}>
-        <span className="description">{slide.text}</span>
-      </div>
+      <>
```
[the remark this answers is not in the document]
Comment (belohlavek) [reply src/components/Modals/TutorialModal/TutorialModal.tsx]: Mmm no error

**Pull request merged**